src/admiral/certs/tasks.py

```diff
@@ -28,63 +28,53 @@
     retry_backoff=True,
     retry_jitter=True,
     retry_kwargs={"max_retries": 16},
+    rate_limit="10/h",
 )
-def summary_by_domain(domain, subdomains=True, expired=False):
+def summary_by_domain(domain, subdomains=True):
     """Fetch a summary of the certificates in the log.
 
     Arguments:
     domain -- the domain to query
     subdomains -- include certificates of subdomains
-    expired -- include expired certificates
 
     """
     # validate input
     m = DOMAIN_NAME_RE.match(domain)
     if m is None:
         raise ValueError(f"invalid domain name format: {domain}")
 
-    wildcard_param = "%." if subdomains else ""
-    expired_param = "" if expired else "&exclude=expired"
+    # fetch api key
+    key = ""
+    with open("/run/secrets/sslmate-api-key.txt", "r") as file:
```

Comment:
"r" is the default mode, so it can be removed here:
```suggestion
    with open("/run/secrets/sslmate-api-key.txt") as file:
```